livingatlas/pipelines/src/main/java/au/org/ala/pipelines/transforms/LocationTransform.java

```diff
@@ -79,6 +84,17 @@ public void setup() {
       log.info("Initialize geocodeKvStore");
       stateProvinceKvStore = stateProvinceKvStoreSupplier.get();
     }
+
+    try {
+      countryCentrePoints = CountryCentrePoints.getInstance(alaConfig.getLocationInfoConfig());
+      stateProvinceCentrePoints =
+          StateProvinceCentrePoints.getInstance(alaConfig.getLocationInfoConfig());
+      stateProvinceVocab =
+          StateProvince.getInstance(alaConfig.getLocationInfoConfig().getStateProvinceNamesFile());
+    } catch (Exception e) {
+      log.error(e.getMessage(), e);
```

Comment:
Shall we provide more hints to help users correct configuration file?

Reply:
@qifeng-bai Im wondering if we need to do this now, as by default the vocab files that are shipped in the JAR will be used unless the user has configured some externally via YAML.

I think when we originally added this help, the vocab files would only be loaded externally so we needed the error messages.

Reply:
I am OK without hints at this moment.